content/docs/compute/index.md

```diff
@@ -12,6 +12,6 @@ description: Launch Instances on Civo's platform in various regions with a range
 
 Civo is a cloud computing platform. As part of the compute services offered, users can launch virtual machines in all available regions. On Civo, these virtual machines are called Instances.
 
-Instances can have a variety of Linux operating systems, and come in a range of sizes and capacities, which determine the price charged to you.
+Instances can have a variety of Linux operating systems, and they come in a range of sizes and capacities. This determines the price charged to you.
```

Comment:
I would leave this sentence as was. Splitting the sentence doesn't seem necessary to me.

Reply:
Noted.